src/Annotation/Controller/Request.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types = 1);
+
+namespace Apitte\Core\Annotation\Controller;
+
+use Doctrine\Common\Annotations\Annotation\Target;
+
+/**
+ * @Annotation
+ * @Target("METHOD")
+ */
+final class Request
+{
+
+	/** @var string|null */
+	private $description;
+
+	/** @var string|null */
+	private $entity;
+
+	/** @var bool */
+	private $required = false;
+
+	/**
+	 * @param mixed[] $values
+	 */
+	public function __construct(array $values)
+	{
+		if (isset($values['description'])) {
+			$this->description = $values['description'];
```

Comment:
`$this->description = $values['description'] ?? null`
Same in response please

Reply:
done